src/content/en/updates/2018/12/badging-api.md

```diff
@@ -15,15 +15,22 @@ description: The Badging API is a new web platform API that allows installed web
 
 <div class="clearfix"></div>
 
-{% include "web/updates/_shared/capabilities.html" %}
+<aside class="caution">
+  We’re currently working on this API as part of the new
+  <a href="/web/updates/capabilities">capabilities project</a>, and starting
+  in Chrome 73 is available as an <a href="#ot"><b>origin trial</b></a>.
+  This post will be updated as the Badging API evolves, and was last updated on
+  January 29th, 2019.
+</aside>
 
 
 ## What is the Badging API? {: #what }
 
 <figure class="attempt-right">
   <img src="/web/updates/images/2018/12/badges.png">
   <figcaption>
-    Examples of badges on launch icons across different platforms.
+    Examples of badges from native apps on launch icons across different
```

Comment:
I left all of the images in and tweaked the text here, since this will likely be supported on all platforms in the future, I feel like it's fair to show other platforms. I tweaked the image caption to mention that these are examples of native apps.

Reply:
We don't have a path forward on Android or Linux at the moment. On Android, you have to show a notification to show a badge and I don't think we're willing to make the compromise at the moment (in future, hopefully Android will add a badging api).

On Linux, we're not super clear if there is an API that's shared across window managers.

On ChromeOS, we're waiting for badging to get implemented on the platform.

But up to you.

Reply:
OK, will leave it in, but I added a short paragraph below that covers supported platforms.